using_images/db_images/mongodb.adoc

```diff
@@ -191,35 +222,33 @@ $ oc env dc mongodb \
   MONGODB_ADMIN_PASSWORD=<new_admin_password>
 ----
 
+[IMPORTANT]
+====
+If multiple deployment configurations utilize these environment variables, for
+example in the case of an application created from a template, you must perform
+the above task on each deployment configuration so that the passwords are in
+sync.
+====
```

Comment:
@rhcarvalho Can you ack the above note? This was my finding after testing the verification steps. For example, I created the dancer-mysql-example template, then had to update the password env vars for both the "dancer-mysql-example" DC and the "database" DC, otherwise they were out of sync. Same for the MongoDB one (haven't got the PostgreSQL yet, but I assume the same applies).